src/styles/blocks/review.scss

```diff
@@ -0,0 +1,15 @@
+.review {
+  display: flex;
+  flex-direction: row;
+  padding: 16px 16px 0;
+  justify-content: space-between;
+  align-items: center;
+
+  &__reviews {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #060b35;
```

Comment:
Font properties can be shortened into a single line using the shorthand property. For example, `font: 400 10px/14px 'Roboto', sans-serif;`. This helps to keep your code concise and improves readability.